generator/Generator/ModelGenerator.php

```diff
@@ -273,8 +278,8 @@ private function processBodySchema(Operation $operation): array
      * @param string $path
      * @param array<array<string,string>> $existingClassHeaders
      * @param array<string,mixed> $pathParameters
-     * @param array<string,mixed> $queryParameters
-     * @param array<\ToshY\BunnyNet\Generator\Model\AbstractParameter> $bodyParameters
+     * @param array<\ToshY\BunnyNet\Model\AbstractParameter> $queryParameters
```

Comment:
nit:

These can now be imported (if not already) as there is only 1 `AbstractParameter` now.